indolaw-nextjs/components/LawPage.tsx

```diff
@@ -26,7 +29,7 @@ export default function LawPage(props: {
             position: fixed;
             padding: 20px;
             width: ${navWidth};
-            background-color: ${colors.background};
+            background-color: ${(isDarkMode ? darkColors : colors).trayBackground};
```

Comment:
@willemchua nitpick: instead of repeating the conditional/ternary operator ```isDarkMode ? darkColors : colors``` multiple times, I think it makes more sense to just do it once at the top of the render function (e.g``` selectedColors = isDarkMode ? darkColors : colors``` and then use ```selectedColors``` everywhere else)

That way if we have to change the dark mode logic in the future we have 1 place to change, instead of having to trackdown all 3 & remember to change them